src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -141,11 +141,15 @@ private bool TryRewriteTagHelper(Block tagBlock, RewritingContext context)
             {
                 // We're in an end tag helper block.
 
-                var tagNameScope = _tagStack.Count > 0 ? _tagStack.Peek().TagName : string.Empty;
+                var parentTagHelper = _tagStack.Count > 0 ? _tagStack.Peek() : null;
```

Comment:
don't need to change this file.  back to using just the `TagName` value (or `null`).